src/test/java/nextstep/subway/line/domain/LineTest.java

```diff
@@ -0,0 +1,120 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.advice.exception.SectionBadRequestException;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.dto.StationResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("Line 도메인 단위 테스트")
+class LineTest {
+
+    private Station 교대역;
+    private Station 강남역;
+    private Station 역삼역;
+    private Station 잠실역;
+
+    private int 교대역_강남역_거리;
+    private int 강남역_역삼역_거리;
+    private int 교대역_역삼역_거리;
+    private int 강남역_잠실역_거리;
+
+    private Line 이호선;
+
+    @BeforeEach
+    void setUp() {
+        교대역 = new Station("교대역");
+        강남역 = new Station("강남역");
+        역삼역 = new Station("역삼역");
+        잠실역 = new Station("잠실역");
+
+        교대역_강남역_거리 = 1;
+        강남역_역삼역_거리 = 3;
+        교대역_역삼역_거리 = 4;
+        강남역_잠실역_거리 = 10;
+
+        이호선 = new Line("2호선", "green", 교대역, 역삼역, 교대역_역삼역_거리);
+    }
+
+    @DisplayName("역의 구간 목록을 순서대로 가져온다")
+    @Test
+    void getStations() {
+        List<StationResponse> 구간_목록 = 이호선.getStationResponse();
+        List<String> 구간_역이름_목록 = 구간_역이름_목록_조회(구간_목록);
+
+        assertThat(구간_역이름_목록).containsExactlyElementsOf(Arrays.asList("교대역", "역삼역"));
+    }
+
+    @DisplayName("역의 구간을 추가한다 : 교대-강남을 추가하여 교대역-강남역-역삼역 순서대로 출력")
+    @Test
+    void addSection() {
+        이호선.addSection(교대역, 강남역, 교대역_강남역_거리);
+
+        List<StationResponse> 구간_목록 = 이호선.getStationResponse();
+        List<String> 구간_역이름_목록 = 구간_역이름_목록_조회(구간_목록);
+
+        assertThat(구간_역이름_목록).containsExactlyElementsOf(Arrays.asList("교대역", "강남역", "역삼역"));
```

Comment:
아래와 같이 사용할 수도 있을 것 같아요~!

```suggestion
        assertThat(구간_역이름_목록).containsExactly("교대역", "강남역", "역삼역");
```

Reply:
오 따로 리스트로 바꾸지 않고도 간결하게 나타낼 수 있네요
감사합니다!